tools/lib/h5tools_utils.c

```diff
@@ -589,12 +589,12 @@ dump_table(hid_t fid, char *tablename, table_t *table)
 
     PRINTSTREAM(rawoutstream, "%s: # of entries = %d\n", tablename, table->nobjs);
     for (u = 0; u < table->nobjs; u++) {
-        H5VLconnector_token_to_str(fid, table->objs[u].obj_token, &obj_tok_str);
```

Comment:
This function is inside a debugging #ifdef, so looks like it was just never updated.